src/bgfx_shader.sh

```diff
@@ -639,7 +643,11 @@ uniform mat4  u_proj;
 uniform mat4  u_invProj;
 uniform mat4  u_viewProj;
 uniform mat4  u_invViewProj;
+#if BGFX_MULTIPLE_TRANSFORMS
```

Comment:
Why not just making `BGFX_CONFIG_MAX_BONES` default to be 1?

Reply:
I removed this stuff from this PR